docs/monitor/enable-notifications.md

```diff
@@ -1,90 +1,7 @@
--   [**Discord**](https://github.com/netdata/netdata/blob/master/src/health/notifications/discord/README.md)
--   [**Dynatrace**](https://github.com/netdata/netdata/blob/master/src/health/notifications/dynatrace/README.md)
--   [**Email**](https://github.com/netdata/netdata/blob/master/src/health/notifications/email/README.md)
--   [**Flock**](https://github.com/netdata/netdata/blob/master/src/health/notifications/flock/README.md)
--   [**Gotify**](https://github.com/netdata/netdata/blob/master/src/health/notifications/gotify/README.md)
--   [**IRC**](https://github.com/netdata/netdata/blob/master/src/health/notifications/irc/README.md)
--   [**Kavenegar**](https://github.com/netdata/netdata/blob/master/src/health/notifications/kavenegar/README.md)
--   [**Matrix**](https://github.com/netdata/netdata/blob/master/src/health/notifications/matrix/README.md)
--   [**Messagebird**](https://github.com/netdata/netdata/blob/master/src/health/notifications/messagebird/README.md)
--   [**Microsoft Teams**](https://github.com/netdata/netdata/blob/master/src/health/notifications/msteams/README.md)
--   [**Netdata Agent dashboard**](https://github.com/netdata/netdata/blob/master/src/health/notifications/web/README.md)
--   [**Opsgenie**](https://github.com/netdata/netdata/blob/master/src/health/notifications/opsgenie/README.md)
--   [**PagerDuty**](https://github.com/netdata/netdata/blob/master/src/health/notifications/pagerduty/README.md)
--   [**Prowl**](https://github.com/netdata/netdata/blob/master/src/health/notifications/prowl/README.md)
--   [**PushBullet**](https://github.com/netdata/netdata/blob/master/src/health/notifications/pushbullet/README.md)
--   [**PushOver**](https://github.com/netdata/netdata/blob/master/src/health/notifications/pushover/README.md)
--   [**Rocket.Chat**](https://github.com/netdata/netdata/blob/master/src/health/notifications/rocketchat/README.md)
--   [**Slack**](https://github.com/netdata/netdata/blob/master/src/health/notifications/slack/README.md)
--   [**SMS Server Tools 3**](https://github.com/netdata/netdata/blob/master/src/health/notifications/smstools3/README.md)
--   [**Syslog**](https://github.com/netdata/netdata/blob/master/src/health/notifications/syslog/README.md)
--   [**Telegram**](https://github.com/netdata/netdata/blob/master/src/health/notifications/telegram/README.md)
--   [**Twilio**](https://github.com/netdata/netdata/blob/master/src/health/notifications/twilio/README.md)
+This section includes the documentation of the integrations for both of Netdata's notification methods.
 
+- Netdata Cloud offers [centralized alert notifications](https://learn.netdata.cloud/docs/alerts-&-notifications/notifications/centralized-cloud-notifications) which leverages the health status information already streamed to Netdata Cloud from connected nodes to send notifications to the configured integrations. Supported integrations are Amazon SNS, Discord, Slack, Splunk and more.
```

Comment:
```suggestion
- Netdata Cloud provides centralized alert notifications, utilizing the health status data already sent to Netdata Cloud from connected nodes to send alerts to configured integrations. [Supported integrations](https://learn.netdata.cloud/docs/alerts-&-notifications/notifications/centralized-cloud-notifications) include Amazon SNS, Discord, Slack, Splunk, and others.
```

Reply:
> health status information already streamed to Netdata

Let's not use the word "stream" anywhere other than streaming area, people might get confused that the Netdata Agent is streaming data to the Netdata Cloud.